How-to: Choose Node Class (#134)
appuio/appuio-cloud-docs: docs/modules/ROOT/pages/references/node-classes.adoc

= Node Classes

Some zones of {product} provide a set of two more node classes.

> A node class is a group of nodes having a distinct set of specidifications.

Reviewer: ```suggestion
A node class is a group of nodes having a distinct set of specifications.
```

Those nodes can be optimized for compute, memory or cost or they could offer special features like GPU support.

The node classes available on a zone are indicated on the https://portal.appuio.cloud/zones[zone listing].

See xref:how-to/choose-node-class.adoc[] on how to schedule your workload on a specific node class.

You will find the characteristics of all node class explained below.

> 

Reviewer: I'd add a heading here:

```suggestion
== cloudscale.ch Zones
```

flex::
+

> Flex nodes directly map to https://www.cloudscale.ch/en/pricing#flavors-explained[flex instances defined by the infrastrucutre provider].

Reviewer: ```suggestion
Flex nodes directly map to https://www.cloudscale.ch/en/pricing#flavors-explained[flex instances defined by the infrastructure provider].
```

They use CPUs shared with other instances.
This allows for short term load peaks but can occasionally require your workload to wait for CPU time.

> Workload running on flex nodes will cost you less (see https://products.docs.vshn.ch/products/appuio/cloud/pricing_2022-12-01.html[pricing]).

Reviewer: ```suggestion
Workload running on Flex nodes will cost you less (see https://products.docs.vshn.ch/products/appuio/cloud/pricing.html[pricing]).
```

That points to the current one.

+
Use this node class for your background jobs, test environments or applications not sensitive to latency.
+
On an {product} zone running on cloudscale.ch, your workload will by default be scheduled on flex nodes.

plus::
+

> Flex nodes directly map to https://www.cloudscale.ch/en/pricing#flavors-explained[plus instances defined by the infrastrucutre provider].

Reviewer: ```suggestion
Plus nodes directly map to https://www.cloudscale.ch/en/pricing#flavors-explained[plus instances defined by the infrastructure provider].
```

Plus nodes have dedicated CPU cores.

> This gives you consistent performance but will cost you more (see https://products.docs.vshn.ch/products/appuio/cloud/pricing_2022-12-01.html[pricing]).

Reviewer: ```suggestion
This gives you consistent performance but will cost you more (see https://products.docs.vshn.ch/products/appuio/cloud/pricing.html[pricing]).
```

+

> The plus node class is probably the right choise for your production frontends, database and other applications sensitive to latency.

Reviewer: ```suggestion
The plus node class is probably the right choice for your production frontends, database and other applications sensitive to latency.
```
